octobot_trading/personal_data/orders/order.py

```diff
@@ -194,9 +198,18 @@ def update(self, symbol, order_id="", status=enums.OrderStatus.OPEN,
         if self.taker_or_maker is None:
             self._update_taker_maker()
 
-        if not self.reduce_only:
+        if reduce_only is not None:
             self.reduce_only = reduce_only
 
+        if allow_self_managed is not None:
+            self.allow_self_managed = allow_self_managed
+
+        if one_cancels_the_other is not None:
+            self.one_cancels_the_other = one_cancels_the_other
+
+        if tag is not None:
+            self.tag = tag
```

Comment:
Are these tags sent to the exchange? Or will they be in the future? Many exchange supports adding tags to orders